test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageControllerSpec.scala

```diff
@@ -3186,13 +3380,31 @@ class MultipleDisposalsTriageControllerSpec
           if (expectReturnToSummaryLink) messageFromMessageKey("returns.return-to-summary-link")
           else ""
         )
+        expectedAdditionalIdKeyValues.map(a => doc.select(a.id).html() should include(a.value))
+        expectedAdditionalNameAttributeKeyValues.map(v =>
+          doc.select(s"${v.tagName}").attr(v.attributeName, v.attributeValue).text() should include(v.value)
+        );
       }
     )
   }
-
 }
 
 object MultipleDisposalsTriageControllerSpec extends Matchers {
+  final case class IdKey(id: String, value: String)
```

Comment:
the way you're using it is more generic than the name suggests - `id` is actually more `selector`. You can't just put the id of the element in this case class the way you're using it.

Reply:
Done.